tools/packaging/packaging.psm1

```diff
@@ -2521,14 +2521,14 @@ function New-MSIPackage
     [Environment]::SetEnvironmentVariable("ProductVersionWithName", $productVersionWithName, "Process")
     if (!$isPreview)
     {
-        [Environment]::SetEnvironmentVariable("PwshPath", '', "Process")
+        [Environment]::SetEnvironmentVariable("PwshPath", "[$productDirectoryName]", "Process")
         [Environment]::SetEnvironmentVariable("UpgradeCodeX64", '31ab5147-9a97-4452-8443-d9709f0516e1', "Process")
         [Environment]::SetEnvironmentVariable("UpgradeCodeX86", '1d00683b-0f84-4db8-a64f-2f98ad42fe06', "Process")
         [Environment]::SetEnvironmentVariable("IconPath", 'assets\Powershell_black.ico', "Process")
     }
     else
     {
-        [Environment]::SetEnvironmentVariable("PwshPath", 'preview', "Process")
+        [Environment]::SetEnvironmentVariable("PwshPath", "[$productDirectoryName]preview", "Process")
```

Comment:
@TravisEz13 Non-blocking comment: Would it not be cleaner if a new variable was declared to not convolute the script with WiX syntax?

Reply:
This variable was already there and only used for this purpose.  I don't think I'm getting the meaning of your statement.

Reply:
I do think there is probably a cleaner way of doing this.  Like refering to the preview directory by name, but the goal here was to make a quick change that would not require a lot of re-testing for the 6.1.0 release.

Reply:
`[$productDirectoryName]preview` is WiX syntax that we inject into the WiX file. The build script should be generic and know as little about WiX as possible to reduce coupling if installer technology changed for example or WiX 4 brings breaking changes in syntax. How about this:
```powershell
[Environment]::SetEnvironmentVariable("PwshPath", (Join-Path $productDirectoryName 'preview'), "Process")
```

Reply:
I'm fine with that change for 6.2.